### PR TITLE
CB2-19285: Update documentation for new data structure

### PR DESCRIPTION
**Pull request by Daniel-Searle**
## Update documentation for new data structure

Our API documentation needs to be updated to be reflective of the changes we’re making to the data structure returned by the IVA defects
[CB2-10285](https://dvsa.atlassian.net/browse/CB2-10285)

## Checklist

- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: docs/defects-api.yml [hunk 1]
```diff
@@ -324,7 +295,10 @@ components:
         inspectionTypes:
           type: array
           items:
-            $ref: "#/components/schemas/inspectionType"
+            type: string
```
Comment (LGin-BJSS) [inline docs/defects-api.yml:298]: can this not still reference the type? if not, can the type be removed?

Comment (LGin-BJSS) [inline docs/defects-api.yml:298]: Can this no still reference the type below? If not, can the `inspectionType` section be removed?

Comment (Daniel-Searle) [reply docs/defects-api.yml:298]: Shout, yes it can

File: docs/defects-api.yml [hunk 2]
```diff
@@ -324,7 +295,10 @@ components:
...
+            enum:
+              - basic
+              - normal
 
     vehicleType:
```
Comment (LGin-BJSS) [inline docs/defects-api.yml:303]: I don't think we are using vehicle type any more, can this be removed?

I think it is also referenced on the `ivaDefect` type and needs to be removed from there.

File: docs/defects-api.yml (hunk 1 above)
Comment (LGin-BJSS) [reply docs/defects-api.yml]: I have no idea why this comment happened twice.

**Review by LGin-BJSS — APPROVED** [review]

**Review by m-coslett — COMMENTED** [review]
Does the ivaDefect object definition at line 263 in the api spec need updating to reflect the euVehicleCategories array and the new basic and normal arrays?

**Review by cb-cs — APPROVED** [review]

**Review by m-coslett — APPROVED** [review]

**Pull request merged**